packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -1315,6 +1315,6 @@ export class AiAgentModel {
     ): Promise<DbAiAgentToolCall[]> {
         return this.database(AiAgentToolCallTableName)
             .where('ai_prompt_uuid', promptUuid)
-            .orderBy('order_index', 'asc');
```

Comment:
what;s the order_index?

Reply:
it was a column that existed on this table before! I had removed it from the migration but kept this method
We now take care of the orderBy via createdAt :D